Description: Try to dlopen curl-gnutls
 Since we link with gnutls, try to dlopen curl-gnutls (in addition to the
 OpenSSL variant).
Author: Andreas Metzler <ametzler@debian.org>
Origin: other
Forwarded: no
Last-Update: 2017-04-23
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/lib/ecore_con/ecore_con_url_curl.c
+++ b/src/lib/ecore_con/ecore_con_url_curl.c
@@ -282,6 +282,9 @@
    if (!_c->mod)
      ERR("Could not find libcurl.5.dylib, libcurl.4.dylib, libcurl.so.5, libcurl.so.4");
 #else
+# ifdef HAVE_GNUTLS
+   LOAD("libcurl-gnutls.so.4"); // try this one first
+# endif /* ifdef HAVE_GNUTLS */
    LOAD("libcurl.so.5"); // try only
    LOAD("libcurl.so.4"); // try only
    if (!_c->mod)
